### PR TITLE
feat(ui): footer

### LINKED ISSUE #267: Identify and Design Footer element
Determine if a footer element is appropriate for App Store, including and not limited to Providing the Unicorn Guarantee.

If valuable, update design. (@Madeline-UX has some mockups)

### PR DESCRIPTION
**Pull request by andrewrisse**
closes #267 

- Added Coming Soon pages for /benefits and /contact so the Footer link test will pass. They will be added very soon and I'd rather have placeholders there than code a workaround for the test and forget to remove it later.
- Scrollbar for app pages will require layout refactor including using routing groups, will do as separate story if desired, but low priority. Scroll bar currently hidden for that page, but it still scrolls (issue has to do with avoiding “Double scrollbar” issue on some pages but not others)

- Tailwind merge package added because I wanted to re-use a style in a component but using a template literal caused prettier issues that broke the code. Added twMerge as a dev dep instead of resolving prettier issue because it is super useful and also to save time. Flowbite Svelte also uses it a lot, so if we need to copy in their components, we will have it.

- Also updated styling of 404 page, view by going to http://localhost:5173/asd/asd (Centered, and “Back to Homepage” link matches styling of footer links”

### CODE REVIEW

File: ui/src/lib/components/Footer.svelte [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      },
+      { title: 'Request An App Not in App Store', href: '/contact' }
+    ],
+    'Defense Unicorns': [
+      { title: 'About Company', href: 'https://www.defenseunicorns.com/', external: true },
+      { title: 'Products', href: 'https://www.defenseunicorns.com/uds', external: true },
+      { title: 'Contact Us', href: 'https://www.defenseunicorns.com/contactus', external: true }
+    ],
+    'For Partners': [
+      { title: 'Contact Vendor Relations', href: 'mailto:sam.k@defenseunicorns.com' }
```
Comment (andrewg-xyz) [inline ui/src/lib/components/Footer.svelte:24]: I think this should be a group email, @gregclark-defenseunicorns // @Madeline-UX // @jonnyborbs is this Sam for a specific reason?

File: ui/src/routes/contact/+page.svelte
Comment (andrewg-xyz) [inline ui/src/routes/contact/+page.svelte:1]: Replace this with an Alert Component, for example: https://github.com/defenseunicorns/uds-appstore/blob/main/ui/src/lib/features/appdetails/Sections/Authorization.svelte

File: ui/src/routes/benefits/+page.svelte
Comment (andrewg-xyz) [inline ui/src/routes/benefits/+page.svelte:1]: Replace this with an Alert Component, for example: https://github.com/defenseunicorns/uds-appstore/blob/main/ui/src/lib/features/appdetails/Sections/Authorization.svelte

File: ui/src/routes/contact/+page.svelte
Comment (andrewrisse) [reply ui/src/routes/contact/+page.svelte:1]: replaced. I modified Alert.svelte as well to allow passing class overrides.

File: ui/src/routes/benefits/+page.svelte
Comment (andrewrisse) [reply ui/src/routes/benefits/+page.svelte:1]: replaced

File: ui/src/lib/components/Footer.svelte [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
```
Comment (andrewg-xyz) [inline ui/src/lib/components/Footer.svelte:24]:
```suggestion
      { title: 'Contact Vendor Relations', href: 'mailto: airgap-appstore@defenseunicorns.com' }
```

**Review by gregclark-defenseunicorns — APPROVED** [review]
Design Review Passed

**Pull request merged**